src/components/UnderConstruction.tsx

```diff
@@ -1,9 +1,42 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Image from "next/image";
 import { FaWhatsapp } from "react-icons/fa";
+import Link from 'next/link';
 import styles from "@/styles/components/UnderConstruction.module.css";
 
 function UnderConstruction() {
+
+  const [days, setDays] = useState(0)
+  const [hours, setHours] = useState(0)
+  const [minutes, setMinutes] = useState(0)
+  const [seconds, setSeconds] = useState(0)
+
+  useEffect(() => {
+    const target = new Date('03/10/2023 23:59:59')
+
+    const interval = setInterval(() => {
+      const now = new Date()
+      const difference = target.getTime() - now.getTime()
+
+      const day = Math.floor(difference / (1000 * 60 * 60 * 24))
+      setDays(day)
+
+      const hours = Math.floor((difference % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60))
+      setHours(hours)
+
+      const minutes = Math.floor((difference % (1000 * 60 * 60)) / (1000 * 60))
+      setMinutes(minutes)
+
+      const seconds = Math.floor((difference % (1000 * 60)) / 1000)
+      setSeconds(seconds)
+
+    }, 1000);
+
+    return () => clearInterval(interval)
+  }, [])
+
+
+  const whatsappUrl = 'https://wa.me/17542441721?text=Hi,%20I%20am%20interesting%20of%20knowing%20more%20about%20your%20services!';
```

Comment:
Fix the message here, it should be "I'm interested in knowing more about your services!"